cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs

```diff
@@ -213,25 +224,4 @@ runAddressBuildScript
   -> NetworkId
   -> Maybe OutputFile
   -> ExceptT ShelleyAddressCmdError IO ()
-runAddressBuildScript (ScriptFile fp) nId mOutFp = do
-  scriptBytes <-
-    handleIOExceptT (ShelleyAddressCmdReadFileException . FileIOError fp) $
-      LBS.readFile fp
-  ScriptInAnyLang _lang script <-
-    firstExceptT (ShelleyAddressCmdAesonDecodeError fp . Text.pack) $
-    hoistEither $
-      Aeson.eitherDecode scriptBytes
-
-  let payCred = PaymentCredentialByScript (hashScript script)
-
-      scriptAddr :: Address ShelleyAddr
-      scriptAddr = makeShelleyAddress nId payCred NoStakeAddress
-                   --TODO: add support for referring to stake addresses
-
-      scriptAddrText :: Text
-      scriptAddrText = serialiseAddress scriptAddr
-
-  liftIO $ case mOutFp of
-    Just (OutputFile oFp) -> Text.writeFile oFp scriptAddrText
-    Nothing               -> Text.putStr        scriptAddrText
-
+runAddressBuildScript scriptFile = runAddressBuild (SourcePaymentScript scriptFile) Nothing
```

Comment:
I'm wondering if we also print a deprecation warning here as well.

Reply:
Do we have an example of a deprecation warning?

Reply:
https://github.com/input-output-hk/cardano-node/blob/0dac653ca71133abc7a5cc01d31d72aabba3d9d9/cardano-cli/src/Cardano/CLI/Parsers.hs#L75

Reply:
Or https://github.com/input-output-hk/cardano-node/blob/94f79cc9d637af569836cd30828681875c2f986a/cardano-api/src/Cardano/Api/Eras.hs#L95